doc/api/async_hooks.md

```diff
@@ -329,6 +329,8 @@ The `type` is a string identifying the type of resource that caused
 `init` to be called. Generally, it will correspond to the name of the
 resource's constructor.
 
+Here is a list of potential _identifiers_:
```

Comment:
This and the other addition in this PR are the only instances of the word _identifiers_ in the document. Is there another word? Why is it italicized? Should it be this maybe?:

```suggestion
Valid values are:
```

Reply:
Thanks Rick for your feedback.

Wasn't sure about the `identifier` word. I remember the word `identifying` inspired me but it's probably not appropriate if you feel it doesn't fit.